### ISSUE TITLE
Lexicon - Avoid box-shadow CSS effect and use a clear outline for focus indicator

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1587]
**Describe the bug**
The focus style for all links (a, nav-inline__link, .dropdown__menu > li > a ), buttons (.btn, .btn–outline), and certain form controls: (radio buttons (.form__control.radio), dataTable sortable headers ( datatable td.sorting:focus) throughout Lexicon uses a background color change to yellow (#fd0) and box shadow effect for an underline and outline:none. Alone, the yellow background shift fails colour contrast against white backgrounds throughout and by using outline: none, we are relying on the box shadow effect for an accessible focus indicator. However, box-shadows may not work as expected in alternative colour schemes so should not be relied upon.

**To Reproduce**
1. Go to Pulsar Lexicon
2. Tab through links in nav bar with keyboard
3. Inspect code to see focus state

**Expected behavior**
Focus indicators should typically be a 2px solid CSS outline that contrasts 3:1, or two tone solid outline to pass against multiple background colours.

**Screenshots**
<img width="421" alt="Image" src="https://github.com/user-attachments/assets/d8176fde-f8e9-4f44-a21b-f6de68addb09" />

<img width="474" alt="Image" src="https://github.com/user-attachments/assets/dd91b9f9-ef38-46e3-a2cb-bb48e855ff1d" />

<img width="381" alt="Image" src="https://github.com/user-attachments/assets/a32ef0d6-5dfd-4e26-b502-b2ce8d0c1b91" />

**Additional context**
Related success criteria:
[1.4.11 Non-text Contrast (AA)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast)
[2.4.7 Focus Visible (AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible)
[2.4.13 Focus Appearance (AAA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance)

**Suggested fixes**
Fix: ​​update the focus indicator style for links to be clear and visible in different colour schemes, passing the requirements of 2.4.13 Focus Appearance (typically a 2px solid CSS outline that contrasts 3:1) with an additional outer box shadow to pass against light and dark backgrounds.

Eg.   
box-shadow: inset 0 0 0 2px #1e1e1e;
outline: 3px solid #fd0;
border-color: #1e1e1e;
background-color: #fd0;   
color: #000;
